File: JQuery/Examples/Add_Tracks.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://api.spotify.com/v1/users/{{user_id}}/playlists/{{playlist_id}}/tracks?uris={{uris}}",
  "method": "POST",
  "headers": {
    "accept": "application/json",
    "authorization": "Bearer {{authorization}}",
    "content-type": "application/json"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: Python/Examples/add_tracks.py
import requests

url = "https://api.spotify.com/v1/users/{{user_id}}/playlists/{{playlist_id}}/tracks"

querystring = {"uris":"{{uris}}"}

headers = {
    'accept': "application/json",
    'authorization': "Bearer {{authorization}}",
    'content-type': "application/json"
    }

response = requests.request("POST", url, headers=headers, params=querystring)

print(response.text)